### PR TITLE
Update documentation to use Timber::context

### PR DESCRIPTION
**Pull request by marciojc**
As Timber::get_context() will be deprecated, I updated the documentation to the new Timber::context()

### CODE REVIEW

**Comment by marciojc** [comment]
@jarednova should I also change the tests?

**Comment by jarednova** [comment]
@marciojc yes please! that'd be great

**Comment by jarednova** [comment]
The changes on `get_context` ==> `context` all look good to me. But I also noticed changes in the `.md` files from spaces to tabs. @gchtr does this present any issues with the 2.x merge?

**Comment by marciojc** [comment]
sorry @jarednova, already fix that and update the editorconfig.

File: docs/guides/wp-integration.md [hunk 1]
```diff
@@ -257,15 +257,15 @@ function get_password_protected_template( $template ) {
     }
 
     return $template;
-}, 99 );
+};
```
Comment (jarednova) [inline docs/guides/wp-integration.md]: whoops! thanks for removing that extra `)`

**Comment by jarednova** [comment]
@marciojc thanks for this edit! On the tabs vs. spaces front in the docs, we're going to do a final pass once Timber 2.0 is done. But before that, we're avoiding stylistic changes like that so we don't create a bunch of potential merge conflicts

**Comment by gchtr** [comment]
> But I also noticed changes in the `.md` files from spaces to tabs. @gchtr does this present any issues with the 2.x merge?

There’s not a "real" issue with tabs for indentation in Markdown files. I just figured that when we use spaces, the indentation is more consistent and reliable. So it’s mainly readability that we’d care about here. The downside is that when people copy code over from the documentation, it will have spaces instead of tabs. But this shouldn’t be a problem if you use a proper editor or IDE ;).

**Pull request merged**